src/ops/aggregate.cc

```diff
@@ -409,7 +409,9 @@ bool Aggregate::measure_operator_cost(Simulator *sim,
   // TODO: implement
   cost_metrics.forward_time = 0.0f;
   cost_metrics.backward_time = 0.0f;
-  cost_metrics.memory_requirement = 0;
+  cost_metrics.inputs_memory = 0;
```

Comment:
Should we also add in an `assert (false)` or throw an exception here to make sure this inaccuracy isn't happening silently?

Reply:
Hmm, what's the assert(false) for? I'm just matching the original implementation to this Aggregate op.

Reply:
If it's not implemented at the moment (I assume this is the case from the `TODO: implement` comment) then adding an `assert` would cause it to fail rather than silently returning an incorrect result. @jiazhihao thoughts?

Reply:
My guess is those were implemented and will be called. I don't know why they were marked as todo implement.